tilequeue/process.py

```diff
@@ -5,15 +5,30 @@
 from tilequeue.tile import coord_to_mercator_bounds
 from tilequeue.tile import pad_bounds_for_zoom
 from tilequeue.tile import tolerance_for_zoom
-from tilequeue.transform import mercator_point_to_wgs84
+from tilequeue.transform import mercator_point_to_lnglat
 from tilequeue.transform import transform_feature_layers_shape
 from tilequeue.transform import calculate_padded_bounds
 from TileStache.Config import loadClassPath
```

Comment:
Ah, good catch. I must have still had it in my path which is why it continued to work for me.

Is it cool to replace that with a different library that only supported loading from a dotted path? One that I've used in the past is [zope.dottedname](https://pypi.python.org/pypi/zope.dottedname).